notebooks/PARSE-KFF-DATA.ipynb

```diff
@@ -0,0 +1,1892 @@
+       "50               Wyoming                                         -   \n",
+       "\n",
+       "0  Free Cost Vaccine When Available  \\\n",
+       "0                                 -   \n",
+       "1                                 -   \n",
+       "2                                 -   \n",
+       "3                                 -   \n",
+       "4                                 -   \n",
+       "5                                 -   \n",
+       "6                                 -   \n",
+       "7                                 -   \n",
+       "8                    State Requires   \n",
+       "9                                 -   \n",
+       "10                                -   \n",
+       "11                                -   \n",
+       "12                                -   \n",
+       "13                                -   \n",
+       "14                                -   \n",
+       "15                                -   \n",
+       "16                                -   \n",
+       "17                   State Requires   \n",
+       "18                   State Requires   \n",
+       "19                   State Requires   \n",
+       "20                   State Requires   \n",
+       "21                   State Requires   \n",
+       "22                                -   \n",
+       "23                                -   \n",
+       "24                                -   \n",
+       "25                                -   \n",
+       "26                                -   \n",
+       "27                                -   \n",
+       "28                   State Requires   \n",
+       "29                                -   \n",
+       "30                                -   \n",
+       "31                                -   \n",
+       "32                   State Requires   \n",
+       "33                                -   \n",
+       "34                                -   \n",
+       "35                                -   \n",
+       "36                                -   \n",
+       "37          State-Insurer Agreement   \n",
+       "38                                -   \n",
+       "39                   State Requires   \n",
+       "40                                -   \n",
+       "41                                -   \n",
+       "42                                -   \n",
+       "43                                -   \n",
+       "44                                -   \n",
+       "45                                -   \n",
+       "46                                -   \n",
+       "47                                -   \n",
+       "48                                -   \n",
+       "49                                -   \n",
+       "50                                -   \n",
+       "\n",
+       "0  State Requires Waiver of Prior Authorization Requirements  \\\n",
+       "0                                                   -          \
```

Comment:
the output should be `output_directory` + `../output/<NAME_OF_THE_NOTEBOOK[_subtable]'